src/app/shared/components/template/components/layout/display-group/display-group.component.ts

```diff
@@ -45,4 +73,38 @@ export class TmplDisplayGroupComponent extends TemplateBaseComponent implements
       return "dashed_box";
     return "default";
   }
+
+  /** Observe the height of the display group wrapper element and update the height of the placeholder accordingly */
+  private initResizeObserver() {
+    if (this.displayGroupWrapper) {
+      this.resizeObserver = new ResizeObserver((entries) => {
+        let containerPadding = 0;
+        // In the case of a sticky header, the top padding/margin of the main app content and template container need to be accounted for,
+        // now that the display group sits at the very top of the content window outside of the main content and template container
+        if (this.params.sticky === "top") {
+          containerPadding = this.getTotalContainerTopPadding();
+        }
+        const entry = entries[0];
+        this.stickyHeight.set(entry.contentRect.height - containerPadding);
+      });
+
+      this.resizeObserver.observe(this.displayGroupWrapper.nativeElement);
```

Comment:
when calling `observer` there should also be some handling for when to `unobserve` to avoid hanging listeners/memory leaks. 
It might be a case that chrome auto-handles this when element destroyed, however I'm not sure if this would still be the case for chrome on android or the ios webview handler, so probably best to manually enforce
https://stackoverflow.com/questions/67581868/should-i-call-resizeobserver-unobserve-for-removed-elements